person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permision: true)
+    @id = Random.rand(1..10)
+    @name = name
+    @age = age
+    @parent_permission = parent_permision
+  end
+
+  def can_use_services?
+    if is_of_age? || @parent_permission
+      true
+    else
+      false
+    end
+  end
```

Comment:
- [ ] Kindly make sure  the public method `can_use_services?` returns true if person `is of age` or if they have `permission from parents`. Right now you are using a method which doesn't exist in your class as `is_of_age?` is different to `of_age?`. Make sure you correct this.